extensions/amp-ad/0.1/amp-ad-ui.js

```diff
@@ -63,8 +63,8 @@ export class AmpAdUIHandler {
     /** {number} */
     this.state = AdDisplayState.NOT_LAID_OUT;;
 
-    /** {?Element} */
-    this.pageProvidedFallback_ = baseInstance.getFallback();
+    /** {!boolean} */
+    this.hasPageProvidedFallback_ = false;
```

Comment:
This can just be set here? `this.hasPageProvidedFallback_ = !!baseInstance.getFallback();`